cfme/tests/infrastructure/test_ssa_vddk.py

```diff
@@ -47,13 +48,16 @@ def ssa_analysis_profile(appliance):
 @pytest.fixture(params=vddk_versions, ids=([item for item in vddk_versions]), scope='function')
 def configure_vddk(request, appliance, provider, vm):
```

Comment:
@sbulage there is the general appliance fixture `configure_fleecing` can that be used here? Or do you use this one for easy parametrization? https://github.com/ManageIQ/integration_tests/blob/e2fa612c8acd84004e51a3743050aaacc2487441/cfme/fixtures/appliance.py#L217-L223

Reply:
Also if vddk 5.5 is failing, do we still need to test it, i.e. is it supported with the versions of vSphere that CFME supports?

Reply:
@john-dupuy yes you are right, but configure_fleecing is generating VDDK urls based on version of provider. Also, I wanted to use parametrization in the tests itself, as it is not used anywhere :)

Reply:
Waiting on PRT for 5.10 before approval, do we still need to test vddk 5.5?

Reply:
@john-dupuy Removed VDDK 5.5 version.